packages/direction-links/src/js/react.js

```diff
@@ -56,12 +56,12 @@ const AUdirectionLink = ({ linkComponent, dark, link, text, direction, className
 		}
 
 		return (
-			<LinkComponent className={ `au-direction-link ${ className } ${ directions[ direction ] }${ dark ? ' au-direction-link--dark' : '' }` } { ...attributeOptions }>{ text }</LinkComponent>
+			<LinkComponent className={ `au-direction-link ${ className } ${ dark ? ' au-direction-link--dark' : '' }` } { ...attributeOptions }>{ text }<span className={ `au-direction-link ${ className } ${ directions[ direction ] }${ dark ? ' au-direction-link--dark' : '' }` } aria-hidden="true"></span></LinkComponent>
```

Comment:
the `<span>` shouldn't have the `au-direction-link` class, since the parent element already has this. lets try and use BEM